src/AsmResolver.DotNet/Cloning/MemberCloner.cs

```diff
@@ -385,7 +385,7 @@ private GenericParameter CloneGenericParameter(MemberCloneContext context, Gener
         {
             var clonedParameter = new GenericParameter(parameter.Name, parameter.Attributes);
 
-            clonedParameter.Number = parameter.Number;
+            //clonedParameter.Number = parameter.Number;
```

Comment:
Remove comment.

Reply:
You can't set the number manually now.

Reply:
Perhaps I should delete the line altogether, is that what you meant? 😛

Reply:
Yes, remove the entire line is what I meant 👍